app/apollo/resolvers/channel.js

```diff
@@ -869,16 +873,18 @@ const channelResolvers = {
       const user = whoIs(me);
 
       try{
-        logger.info({ req_id, user, org_id, uuid }, `${queryName} validating`);
-
         validateString( 'org_id', org_id );
         validateString( 'uuid', uuid );
 
         const channel = await models.Channel.findOne({ uuid, org_id });
         if(!channel){
           throw new NotFoundError(context.req.t('Channel uuid "{{channel_uuid}}" not found.', {'channel_uuid':uuid}), context);
         }
+
+        logger.info({req_id, user, org_id, uuid}, `${queryName} validating`);
```

Comment:
This log line shouldn't be moved from its prior location just after the try{ line -- the lines between there and here are also 'validation'.